packages/react-components/react-table/library/src/hooks/types.ts

```diff
@@ -166,7 +166,7 @@ export interface UseTableFeaturesOptions<TItem> {
   getRowId?: (item: TItem) => TableRowId;
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33462&commitId=ba8718734224f44c1c8d402c977d2ac0ba4369f6&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|12| Changed|
|Avatar Converged.badgeMask.chromium.png|15| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay High Contrast.chromium.png|4930| Changed|
|Drawer.Full Overlay RTL.chromium.png|1172| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>